tmt/steps/report/junit.py

```diff
@@ -103,6 +47,19 @@ class ReportJUnitData(tmt.steps.report.ReportStepData):
         help='Path to the file to store JUnit to.',
         normalize=lambda key_address, raw_value, logger: Path(raw_value) if raw_value else None)
 
+    flavor: str = field(
+        default=None,
+        option='--flavor',
+        metavar='FLAVOR',
+        choices=[DEFAULT_FLAVOR_NAME],
+        help="Name of a JUnit flavor to generate. By default, the 'default' flavor is used.")
+
+    template_path: str = field(
+        default=None,
```

Comment:
When the default is `None`, type of the field cannot be just `str` - when left unset, `None` would be assigned to it & we'd get apparently string field with the value of `None`. `Optional[str]` would be the right type here and above.